source/api_documentation/signal_api/index.rst

```diff
@@ -107,19 +107,19 @@ Used to create the fields of a call outcome.
 
 Endpoint:
 
-``https://invoca.net/api/@@CALL_OUTCOME_VERSION/call_outcomes``
+``https://invoca.net/api/@@SIGNAL_API_VERSION/transactions/signals``
 
 .. api_endpoint::
   :verb: POST
-  :path: /call_outcomes
-  :description: Create a call outcome
-  :page: create_call_outcome
+  :path: /transcations/signals
+  :description: Create a signal
+  :page: create_signal
```

Comment:
@oorion I got it straightened out but it looks like the directory was renamed to `signal_api` but the partials were not renamed during this merge (and the other one to 2015-12-10) so the `.. api_endpoint::` directive couldn't find what it was looking for. Unfortunately the build script doesn't even raise a warning about this so we need to keep on our toes when renaming partials.